data/players.yaml

```diff
@@ -3613,3 +3613,8 @@
   liquipedia: Valak
   twitch: https://www.twitch.tv/imvalak
   team: QueNaNina
+  - name: Ripvanwinkle
+  country: tr
+  platforms:
+    de:
+    - '1820205'
```

Comment:
You can add your other accounts to the list here. Also if you played on Voobly you might add that as well.